tests/e2e/nativeCommands/adbTypeText.ts

```diff
@@ -1,7 +1,8 @@
 import execAsync from '../utils/execAsync';
 import * as Logger from '../utils/logger';
 
-const adbTypeText = async (text) => {
+// eslint-disable-next-line @typescript-eslint/require-await
+const adbTypeText = async (text: string) => {
```

Comment:
Same, see [the comment](https://github.com/Expensify/App/pull/37676#discussion_r1511474451)